tests/analyze-fixtures/Statement/DoNotUseGlobals.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Tests\Analyze\Fixtures\Statement;
+
+$a = 1;
+
+class DoNotUseGlobals {
+
+    /**
+     * test global
+     */
+    public function testGlobal()
+    {
+        global $a;
+    }
+
+    /**
+     * test no global
+     */
+    public function testNoGlobal()
+    {
+        $a = 2;
+
+        return $a * 3;
+    }
+
+}
+
+?>
+----------------------------
+[{
+    "type": "missing_docblock",
```

Comment:
not sure why I'm getting these errors, should be the missing global check the one triggered shouldn't be?